static/app/views/issueList/actions/sortOptions.tsx

```diff
@@ -53,14 +53,10 @@ const IssueListSortOptions = ({onSelect, sort, query, hasPageFilters}: Props) =>
 
   return (
     <StyledDropdownControl
-      buttonProps={
-        hasPageFilters
-          ? {
-              size: 'xsmall',
-              icon: <IconSort size="xs" />,
-            }
-          : {prefix: t('Sort by')}
-      }
+      buttonProps={{
+        size: 'xsmall',
+        icon: <IconSort size="xs" />,
+      }}
       menuWidth={hasPageFilters ? '150px' : undefined}
```

Comment:
do we need to remove these occurrences too?

Reply:
Oh right, completely missed those! Removed them in https://github.com/getsentry/sentry/pull/34019/commits/1a92f81c27837a6f68fae58c9d75e68d345ba376.